readme.Rmd

```diff
@@ -180,6 +181,15 @@ devtools::install_github("GraceYoon/SPRING")
 devtools::install_github("zdk123/SpiecEasi")
 ```
 
+You can also install NetCoMi from conda bioconda channel with
```

Comment:
```suggestion
### Bioconda

Thanks to [@daydream-boost](https://github.com/daydream-boost), NetCoMi can also 
be installed from conda bioconda channel with
```

Reply:
Yes. I have updated the readme. Thank you for your review.